mac/README.md

```diff
@@ -6,47 +6,30 @@ Xcode 8.3.3 (it might also work to use an older version)
 ## Keyman for macOS Development
 Keyman for macOS can be built from a command line (preferred) or Xcode.
 
-### Compiling from Command line
+### Compiling from Command Line
 To build Keyman for macOS, do the following:
-1. Open a Terminal window.
-2. Change to please run the build.sh build script within this folder. Run
-with the -help switch to see all options.
-You will need to either run with the command-line argument -no-codesign or obtain the
-necessary security permissions and certificate data as part of our core development team
-to complete the build for the final app.
-To build using Xcode, you will need to build KeymanEngine4Mac first and then build
-Keyman4MacIM.
+1. Open a terminal
```

Comment:
Terminal is the proper name of the program, so it should say "Open a Terminal window." or "Open a window in Terminal."

Reply:
Terminal is a program but any generic terminal (eg. iTerm2) would work so that's why I had it as lowercase.